src/main/scala/dev/atedeg/Configuration.scala

```diff
@@ -0,0 +1,119 @@
+package dev.atedeg
+
+import scala.util.Try
+
+import better.files.File
+import io.circe.parser.{ parse => parseJsonString }
+import io.circe.{ Decoder, Json }
+import io.circe.yaml.parser
+import cats.implicits.*
+
+import Extensions.*
+
+sealed trait IgnoredSelector
+final case class IgnoredClass(className: String) extends IgnoredSelector
+final case class IgnoredTrait(traitName: String) extends IgnoredSelector
+final case class IgnoredEnum(enumName: String) extends IgnoredSelector
+final case class IgnoredType(typeName: String) extends IgnoredSelector
+final case class IgnoredEnumCase(caseName: String) extends IgnoredSelector
+
+object IgnoredSelector {
+
+  def fromKind(name: String, kind: String): Option[IgnoredSelector] = kind match {
+    case "class" => Some(IgnoredClass(name))
+    case "trait" => Some(IgnoredTrait(name))
+    case "type" => Some(IgnoredType(name))
+    case "enum" => Some(IgnoredEnum(name))
+    case "case" => Some(IgnoredEnumCase(name))
+    case _ => None
+  }
+}
+
+sealed trait Selector {
+
+  def toIgnored: IgnoredSelector = this match {
+    case Class(className) => IgnoredClass(className)
+    case Trait(traitName) => IgnoredTrait(traitName)
+    case Enum(enumName) => IgnoredEnum(enumName)
+    case Type(typeName, _) => IgnoredType(typeName)
+    case EnumCase(caseName, _) => IgnoredEnumCase(caseName)
+  }
+}
+final case class Class(className: String) extends Selector
+final case class Trait(traitName: String) extends Selector
+final case class Enum(enumName: String) extends Selector
+final case class Type(typeName: String, lookupFile: String) extends Selector
+final case class EnumCase(caseName: String, lookupFile: String) extends Selector
+
+final case class Configuration(ignored: Set[IgnoredSelector], tables: List[TableConfig])
+
+final case class TableConfig(
+    name: String,
+    termName: Option[String],
+    definitionName: Option[String],
+    rows: List[Selector],
+)
+
+private object Utils {
+
+  private def openFile(file: File): Either[Error, String] =
+    Try(file.contentAsString).toEither.leftMap(ExternalError)
+
+  def parseFileWith[A](file: File)(parser: String => Either[Error, A]): Either[Error, A] =
+    openFile(file).flatMap(parser)
+}
+
+object AllEntities {
+  private val entitiesFileName = "searchData.js"
+
+  private def allEntitiesFile(workingDir: File): File =
+    workingDir / "target" / "site" / "scripts" / entitiesFileName
```

Comment:
Hardcoded path. In unidoc is possible to override the destination path, in this case this path is invalid.